### PR TITLE
Skip Python2 support and require python 3.6 or higher

### PR DESCRIPTION
**Pull request by adybbroe**
Skip Python2 support and require python 3.6 or higher.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -32,8 +32,8 @@
       cmdclass=versioneer.get_cmdclass(),
       description='String parser/formatter',
       long_description=README,
-      author='Panu Lahtinen',
-      author_email='panu.lahtinen@fmi.fi',
+      author='The Pytroll Team',
+      author_email='pytroll@googlegroups.com',
```
Comment (mraspaud) [inline setup.py:36]: @adybbroe the problem with this is that only people subscribed to the mailing list can send an email there, right ?

Comment (mraspaud) [reply setup.py:36]: @adybbroe what do you think we should do here?

Comment (djhoese) [reply setup.py:36]: I think we started having this conversation a year ago when this PR was first made. We never really came to a solution. We could make a `groups.io` maintainer email list? Or designate someone as **the** core maintainer of a package and put their email.

Comment (mraspaud) [reply setup.py:36]: How about using slack? https://slack.com/intl/en-se/help/articles/206819278-Send-emails-to-Slack

Comment (djhoese) [reply setup.py:36]: That's an interesting idea. A little scary, but interesting. If you want to try it on one of the pytroll packages and see how long it takes to get spam then :+1:

File: setup.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 #
-# Copyright (c) 2014, 2015
+# Copyright (c) 2014, 2015, 2020
```
Comment (djhoese) [inline setup.py:4]:
```suggestion
# Copyright (c) 2014-2022 trollsift developers
```

File: setup.cfg [hunk 3]
```diff
@@ -2,7 +2,7 @@
 description-file = README.md
 
 [bdist_rpm]
-requires=python python-six
+requires=python3-six
```
Comment (djhoese) [inline setup.cfg:5]: `six` isn't a dependency on this package. Why is this needed?

File: setup.py [hunk 4]
```diff
@@ -32,8 +32,8 @@
       cmdclass=versioneer.get_cmdclass(),
       description='String parser/formatter',
       long_description=README,
```
Comment (djhoese) [inline setup.py:34]:
```suggestion
      long_description=README,
      long_description_content_type='text/markdown',
```

File: setup.py (hunk 1 above)
Comment (adybbroe) [reply setup.py:36]: I would leave it with the pytroll google group, that is how we have in Satpy, and it minimizes the risk of spam. Anyhow, it is really not difficult to get in touch with Pytroll, either via slack or mail.
Ok @djhoese and @mraspaud ?

File: setup.cfg (hunk 3 above)
Comment (adybbroe) [reply setup.cfg]: You are right, don't see why it should be there - removed it!

File: setup.py (hunk 4 above)
Comment (adybbroe) [reply setup.py:34]: Ahh, nice, yes!

**Review by mraspaud — APPROVED** [review]
LGTM
